configs/components/_base-module.rb

```diff
@@ -39,8 +39,13 @@
   pkg.directory(settings[:module_vendordir], mode: '0755', owner: 'root', group: 'root')
 end
 
+target_directory = File.join(settings[:module_vendordir], module_name)
+
 pkg.install do
   [
-    "mv #{module_author}-#{module_name} #{File.join(settings[:module_vendordir], module_name)}"
+    # Rename appropriately for use with puppet
+    "mv #{module_author}-#{module_name} #{target_directory}",
+    # Remove git and CI-related files
+    "rm -r #{File.join(target_directory, '.*')} 2>/dev/null || true",
```

Comment:
Is this executed in a shell context such that the "." is a literal?

Reply:
It doesn't remove non-hidden files, so it appears so -- I'll post some directory listings once the jenkins tests finish

Reply:
Ya know, upon a moment's reflection, thats a silly question. If there was *anything* left in the directory, then  it was operating with shell globbing.

Reply:
Why `|| true`? If a file in `.git` is read-only with the `rm` fail?

Reply:
This will fail the vanagon build when rm is unable to remove `.` or `..` unless it returns successfully. Although, come to think of it, it would be better to avoid this workaround completely -- I'll see what else will work on all our platforms.

Reply:
How about `find . -mindepth 1 -maxdepth 1 -name '.*' | xargs rm -f`

Reply:
hmm looks like on solaris at least `mindepth` and `maxdepth` are not a thing -- `rm -r .[!.]*` seems like it may work though